docs/services/slam/cartographer/_index.md

```diff
@@ -1,20 +1,23 @@
 ---
-title: "Cartographer Integrated Library"
+title: "Cartographer Modular Resource"
 linkTitle: "Cartographer"
 weight: 70
 type: "docs"
-description: "Configure a SLAM service with the Cartographer library."
+description: "Configure a SLAM service with the Cartographer modular resource."
 tags: ["slam", "services"]
+icon: "/services/img/icons/slam.svg"
 aliases:
   - "/services/slam/run-slam-cartographer"
 # SMEs: Kat, Jeremy
 ---
 
 [The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data.
 
+To easily integrate Cartographer into the Viam ecosystem, use the [`viam-cartographer`](https://github.com/viamrobotics/viam-cartographer) library, which wraps Cartographer as a [modular resource](/program/extend/modular-resources/). `viam-cartographer` provides the `cartographer-module` module, which includes the `viam:slam:cartographer` custom SLAM service [model](/program/extend/modular-resources/#models).
```

Comment:
```suggestion
To easily integrate Cartographer into the Viam ecosystem, use the [`viam-cartographer`](https://github.com/viamrobotics/viam-cartographer) library, which wraps Cartographer as a [modular resource](/program/extend/modular-resources/). 
`viam-cartographer` provides the `cartographer-module` module, which includes the `viam:slam:cartographer` custom SLAM service [model](/program/extend/modular-resources/#models).
```

Reply:
🤦 Thank you!

Reply:
(opt) It might be reasonable to try to combine this into one sentence if you could, these sentences are still a bit awkward sounding but i think it's inevitable with this situation 😆

Reply:
Totally -- I tried a bunch! I think I'm gonna go with what we've got while I keep learning the intricacies of our modular approach. Will revisit once I've got a better grasp!